src/kaleidoscope/hid.cpp

```diff
@@ -156,10 +161,19 @@ void initializeAbsoluteMouse() {
 }
 
 void moveAbsoluteMouse(signed char x, signed char y, signed char wheel) {
-  SingleAbsoluteMouse.move(x, y, wheel);
+  moveAbsoluteMouseWithHorizontalWheel(x, y, wheel, 0);
+}
+
+void moveAbsoluteMouseWithHorizontalWheel(signed char x, signed char y, signed char vWheel, signed char hWheel) {
+  SingleAbsoluteMouse.move(x, y, vWheel, hWheel);
 }
+
 void moveAbsoluteMouseTo(uint16_t x, uint16_t y, signed char wheel) {
-  SingleAbsoluteMouse.moveTo(x, y, wheel);
+  moveAbsoluteMouseWithHorizontalWheelTo(x, y, wheel, 0);
+}
+
+void moveAbsoluteMouseWithHorizontalWheelTo(uint16_t x, uint16_t y, signed char vWheel, signed char hWheel) {
+  SingleAbsoluteMouse.moveTo(x, y, vWheel, hWheel);
 }
```

Comment:
See my previous note, these functions can be combined into one while still preserving the old API, if you use default arguments. Thus, only `moveAbsoluteMouse(x, y, vWheel = 0, hWheel = 0)` and `moveAbsoluteMouseTo(x, y, vWheel = 0, hWheel = 0)` would remain.